src/components/GlobalPreferences/Notifications/SubscriptionsTable.js

```diff
@@ -67,20 +71,20 @@ const SubscriptionsTable = React.memo(function SubscriptionsTable({
   }, [])
   const onOrganizationChange = useCallback(idx => {
```

Comment:
All of these handlers should reset the pagination no? Is there pagination actually? We might be missing it.

Reply:
Pagination is included by `DataView` by default, and if you don't control it from outside, it manages its own state. It does not yet reset the pagination when the items change, but discussing if it should.

Reply:
> discussing if it should?
I do think it should, it would be very weird (and I saw it that's why I manually reset pages on other filters) for the user to apply a filter and not see results because of being on the wrong page.